### PR TITLE
[zh] translate concepts.md

### PR DESCRIPTION
**Pull request by SAMZONG**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+- 在使用默认策略的情况下，用户可以直接与 Kubernetes API 交互。
 
-### Override Policy
+## 差异化策略
 
-Karmada provides a standalone Override Policy API for specializing the automation of cluster-related configuration. For example:
+Karmada 为不同的集群提供了一个可自动化生产独立配置的 Override Policy API。例如：
 
-- Override the image prefix based on the member cluster region.
+- 基于成员群集所在区域自动配置不同镜像仓库地址。
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md:23]: 群集 -> 集群

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+本页面介绍了有关 Karmada 的一些核心概念。
 
-### Resource Template
+## 资源模板
 
-Karmada uses the Kubernetes Native API definition for the federated resource template, to make it easy to integrate with existing tools that have already been adopted by Kubernetes.
+Karmada 使用 Kubernetes 原生 API 定义联邦资源模板，以便轻松与现有 Kubernetes 采用的工具进行集成。
 
-### Propagation Policy
+## 传播策略
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md:11]: 所有传播 -> 调度/分发。传播在 Karmada 的语境中不是一个好词语。

**Review by Poor12 — COMMENTED** [review]
I'd like to invite @windsonsea to help review it.

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md (hunk 2 above)
Comment (SAMZONG) [reply i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md:11]: 是否称为部署策略呢？ 我看在不少分享 karmada 的文章内，一般把 pp 解释为 部署策略

**Review by windsonsea — COMMENTED** [review]
Overall the text is good, but please consider making the wording more consistent with [glossary](https://karmada.io/zh/docs/reference/glossary/). It is worth mentioning that some sentences are so good beyond translation 👍

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md (hunk 2 above)
Comment (SAMZONG) [reply i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md:11]: 投一票作为 >> 调度

**Comment by SAMZONG** [comment]
@RainbowMango @windsonsea please review, thanks a lot.

`WHAT I DO`: #386 already merged. rebase branch and sync to v1.6.

```
./i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/concepts.md
./i18n/zh/docusaurus-plugin-content-docs/version-v1.6/core-concepts/concepts.md
```

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**